src/NexusMods.StandardGameLocators/SteamLocator.cs

```diff
@@ -46,6 +48,21 @@ protected override IGameLocatorResultMetadata CreateMetadata(SteamGame game)
             AppId = game.AppId.Value,
             ManifestIds = game.AppManifest.InstalledDepots.Select(x => x.Value.ManifestId.Value).ToArray(),
             CloudSavesDirectory = game.GetCloudSavesDirectoryPath(),
+            ProtonPrefixDirectory = game.GetProtonPrefix()?.ProtonDirectory,
+            GetLaunchOptions = () =>
+            {
+                var localConfigPath = SteamLocationFinder.GetUserDataDirectoryPath(game.SteamPath, game.AppManifest.LastOwner).Combine("config").Combine("localconfig.vdf");
+                var parserResult = LocalUserConfigParser.ParseConfigFile(game.AppManifest.LastOwner, localConfigPath);
+                if (parserResult.IsFailed)
+                {
+                    Logger.LogWarning("Error while parsing local user config at `{Path}`: `{Error}`", localConfigPath, parserResult.Errors);
+                    return null;
+                }
+
+                if (!parserResult.Value.LocalAppData.TryGetValue(game.AppId, out var localAppData)) return null;
```

Comment:
Worth noting this is not available until the game is booted at least once.
If we don't do it, someone else will; not a big issue as App would issue diagnostic after the user closes the game, but it's there.